### PR TITLE
Add pytest script for logging module

### PR DESCRIPTION
**Pull request by zzzzwj**

### CODE REVIEW

File: torchensemble/tests/test_logging.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import pytest
+from torchensemble.utils.logging import set_logger
+
+
+def record(logger):
```
Comment (xuyxu) [inline torchensemble/tests/test_logging.py:5]: Is this a private funciton? If so, how about changing its name to `_record`.

File: torchensemble/tests/test_logging.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    record(logger)
+    logger = set_logger("Loglevel_WARNING", "WARNING")
+    record(logger)
+    logger = set_logger("Loglevel_ERROR", "ERROR")
+    record(logger)
+    logger = set_logger("Loglevel_CRITICAL", "CRITICAL")
+    record(logger)
+
+    with pytest.raises(ValueError):
+        set_logger("Loglevel_INVALID", "INVALID")
```
Comment (xuyxu) [inline torchensemble/tests/test_logging.py:26]: How about adding another line that makes sure that the error message is exactly what we expected. Something like https://github.com/xuyxu/Ensemble-Pytorch/blob/master/torchensemble/tests/test_set_optimizer.py#L44

**Review by xuyxu — COMMENTED** [review]
Good work @zzzzwj. I have left some comments, and let me know if you have any problem on fixing them.

**Comment by zzzzwj** [comment]
OK, I'm working on it now.

**Comment by xuyxu** [comment]
Use another PR to avoid appveyor CI

**Pull request closed without merging**